scripts/run_mainnet.sh

```diff
@@ -4,6 +4,8 @@ set -e
 
 sudo apt update
 sudo apt install -y git clang curl libssl-dev llvm libudev-dev pkg-config
+sudo apt install -y protobuf-compiler
+sudo apt install cmake g++ make
```

Comment:
Append them on the same command on line 6.

Reply:
@zycon91

Reply:
> Append them on the same command on line 6.

sorry for delay. It is done.